index.html

```diff
@@ -26,18 +26,43 @@
             </a>
         </nav>
     </header>
-    <div class="container">
-        <main>
+    <div class="container flex-container">
+        <main class="flex-container">
             <section>
                 <div class="name">
                     <h1>Felix Odete</h1>
                 </div>
-                <div class="about-me">
+                <div class="about-me flex-container">
                     <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
                         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
                 </div>
-                <div class="rectangle-button">
-                    <h1>Start collaboration</h1>
+                <a href="">
+                    <div class="rectangle-button">
+                        <h1>Start collaboration</h1>
+                    </div>
+                </a>
+                <div class="middle-image"></div>
+            </section>
+            <section>
+                <h2 class="projects">Projects</h2>
+            </section>
+            <section class="computer-image"><h2></h2></section>
+            <section class="project-section">
+                <h2>Project name goes here</h2>
+            </section>
+            <section class="grid-container">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
```

Comment:
- [ ] Please Implement all the work sections as in [Figma]( https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)

![image](https://user-images.githubusercontent.com/74744787/158980900-ece2880b-e339-46cb-b20c-896b96b10606.png)

Reply:
Hi! Wrapped this in a `<section>` tag.